test-zeekoe.py

```diff
@@ -144,16 +144,16 @@ def list_channels(cust_config):
     cmd = ["./target/debug/zkchannel", "customer", "--config", cust_config, "list"]
     return run_command(cmd, True)
 
+def expire_channel(merch_config, channel_id, verbose):
+    info("Initiate expiry on the channel id: %s" % channel_id)
+    cmd = ["./target/debug/zkchannel", "merchant", "--config", merch_config, "close", "--channel", channel_id]
+    return run_command(cmd, verbose)
```

Comment:
What do you think of wrapping `zkchannel`? In other words:

```python
def zkchannel_merchant(command, options = {}):
    # ...

def zkchannel_customer(command, options = {}):
    # ...

# use like:
zkchannel_merchant("close", config = merch_config, channel = channel_id)
```

If you do that, I don't even think you need e.g. `expire_channel` because you can just call `zkchannel_merchant("close", ...)` directly.

Reply:
Opted to do this in a followup after chatting in Slack

Reply:
created issue #292 for it